EIPS/EIP-xxx.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: <to be assigned>
+title: EIPS eligible for Inclusion
+author: James Hancock (MadeofTin)
+discussions-to: <URL>
+status: Draft
```

Comment:
Didn't we say this is in Active status (e.g. a registry)?

Reply:
Should it go directly to Active? Or start as Draft and then be moved to Active?